src/stores/models/Finalize.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Model for Finalize.
+ */
+export default class Finalize {
```

Comment:
I know why you used `Finalize` instead of `ResultSet`, but we should still use the name of the DB layer model `ResultSet`. If we ever used `ResultSet` later on, we would use the same model so the Finalize model does not fit the naming for it.